client/src/app/site/motions/modules/motion-detail/components/base/base-motion-detail-child.component.ts

```diff
@@ -0,0 +1,281 @@
+import { Directive, Input } from '@angular/core';
+
+import { Subscription } from 'rxjs';
+import { filter } from 'rxjs/operators';
+
+import { MotionBlockRepositoryService } from 'app/core/repositories/motions/motion-block-repository.service';
+import { MotionCategoryRepositoryService } from 'app/core/repositories/motions/motion-category-repository.service';
+import { MotionChangeRecommendationRepositoryService } from 'app/core/repositories/motions/motion-change-recommendation-repository.service';
+import { MotionLineNumberingService } from 'app/core/repositories/motions/motion-line-numbering.service';
+import { MotionRepositoryService } from 'app/core/repositories/motions/motion-repository.service';
+import { MotionStatuteParagraphRepositoryService } from 'app/core/repositories/motions/motion-statute-paragraph-repository.service';
+import { MotionWorkflowRepositoryService } from 'app/core/repositories/motions/motion-workflow-repository.service';
+import { MotionService } from 'app/core/repositories/motions/motion.service';
+import { TagRepositoryService } from 'app/core/repositories/tags/tag-repository.service';
+import { UserRepositoryService } from 'app/core/repositories/users/user-repository.service';
+import { ComponentServiceCollector } from 'app/core/ui-services/component-service-collector';
+import { ViewUnifiedChange } from 'app/shared/models/motions/view-unified-change';
+import { BaseComponent } from 'app/site/base/components/base.component';
+import { ViewMotion } from 'app/site/motions/models/view-motion';
+import { ViewMotionChangeRecommendation } from 'app/site/motions/models/view-motion-change-recommendation';
+import { ChangeRecoMode, LineNumberingMode } from 'app/site/motions/motions.constants';
+import { MotionServiceCollectorService } from '../../../services/motion-service-collector.service';
+import { MotionViewService } from '../../../services/motion-view.service';
+
+@Directive()
+export class BaseMotionDetailChildComponent extends BaseComponent {
```

Comment:
base class directive should probably be abstract if you never plan to actually make it an instance or a component